src/main/java/com/game/Battleship/models/ComputerBoard.java

```diff
@@ -0,0 +1,5 @@
+package com.game.Battleship.models;
+
+public class ComputerBoard extends Board{
```

Comment:
I like that this is inheriting from the Board class, but is there currently any difference in behavior? Can you explain your design choice to add the computer board right now if its not any different than the regular board?

Reply:
The patch request when adding ships will update the playerBoard and the patch request  when hit the board will  update the ComputerBoard.